eth/polls/polltx.go

```diff
@@ -35,6 +37,9 @@ func (p *EthTxPoll) Poll(
 	if p.lastBlockRead+1 > uint64(state.Block().Height) {
 		return p, nil, nil
 	}
+	if uint64(state.Block().Height)-p.lastBlockRead > p.maxBlockRange {
+		p.lastBlockRead = uint64(state.Block().Height) - p.maxBlockRange
```

Comment:
Here too polling would end up missing blocks, the end of the range should be set to `p.lastBlockRead + maxBlockRange`

Reply:
Done